### DISCUSSION TITLE
Let's create a voting feature!

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5801] [Ideas]
Hi, I am looking to create an add-on to add a voting feature for listeners.

For the moment, it will use the Azuracast nowplaying API (as I had previously done to make it possible to add images with the twitter weebhook) with an external database.

Do you have any ideas for features you would like to see with this voting feature?

**Comment by JulianBorghuis** [comment]
Really nice idea i hope that you get it, can you add me on Discord: Julian#8969 and notify me about this? So I can beta test also?

**Comment by Mozitux** [comment]
Hi, voting feature is ready for testing.
If there are any volunteer's guinea pigs, I can let them test the features before publishing the addon. Here is what you will need:

- A web radio using Azuracast and whose NP API works correctly.
- The possibility of creating an independent MySQL database.
- An independent web server using Apache2 with PHP dependencies (not tested on Nginx).

**Reply by JulianBorghuis** [reply]
I want to test

**Comment by BusterNeece** [comment]
@Mozitux Why not submit this as a Pull Request to AzuraCast itself, rather than implement it as a completely separate project?

**Reply by Mozitux** [reply]
Unfortunately, I don't know enough how Azuracast works to implement it directly, my dev knowledge and skills are also limited at the moment
